WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewModel.swift

```diff
@@ -51,7 +57,7 @@ final class DashboardViewModel: ObservableObject {
 
     var siteURLToShare: URL? {
         if let site = stores.sessionManager.defaultSite,
-           site.isPublic, // only show share button if the site is public.
+           !site.isWordPressComStore || site.isPublic, // only show share button if it's a .org site or a public .com site
```

Comment:
From what I understand, .org sites do not by default have an "unlaunched" setting, unlike .com sites. You'd need a plugin for such "coming soon" functionality (e.g:[ check this guide](https://www.sitesaga.com/hide-wordpress-site-until-ready/), which does not mention any core functionality to do so)

The `blog_public` setting in a .org site (which I think translates to `isPublic` via PR #9480), has 2 options:

> blog_public:
> 1 : I would like my blog to be visible to everyone, including search engines (like Google, Sphere, Technorati) and archivers. (default)
> 0 : I would like to block search engines, but allow normal visitors.

([Source from documentation](https://developer.wordpress.org/apis/options/#privacy))

Thus, I think we should still allow sharing a .org site where `blog_public`, since it still allows normal visitors to visit the site. What do you think?

Reply:
TIL, thanks for the explanation!